templates/zerver/app/invite_user.html

```diff
@@ -37,14 +37,14 @@ <h3 id="invite-user-label">{% trans %}Invite users to Zulip{% endtrans %}</h3>
                         </select>
                     </div>
                 </div>
-                <div class="alert" id="invite_status"></div>
-                {% if development_environment %}
-                <div class="alert" id="dev_env_msg"></div>
-                {% endif %}
                 <div class="control-group">
                     <label class="control-label">{{ _('Streams they should join') }}</label>
                     <div class="controls" id="streams_to_add"></div>
                 </div>
+                <div class="alert" id="invite_status"></div>
+                {% if development_environment %}
+                <div class="alert" id="dev_env_msg"></div>
+                {% endif %}
```

Comment:
I think this change is a mistake -- if there are more than a certain number of streams, you won't see the message.  Better to put it at the top.